server/services/socket.js

```diff
@@ -76,5 +79,10 @@ const promptOtherPlayers = currentPlayer => sockets.filter(socket => socket.hand
 exports.promptOtherPlayers = promptOtherPlayers;
 
 // Ask the other players to vote on the cards
-const promptPlayersVote = currentPlayer => sockets.filter(socket => socket.handshake.session.user !== currentPlayer.username).forEach(socket => socket.emit("vote"));
+const promptPlayersVote = currentPlayer => {
+    sockets.filter(socket => socket.handshake.session.user !== currentPlayer.username)
+    .forEach(socket => socket.emit("vote"));
```

Comment:
It's not going to make much difference but you don't need to use `filter` _and_ `forEach` here. E.g.:
`sockets.forEach(socket => socket.handshake.session.user !== currentPlayer.username && socket.emit("vote"));`
Is that better? 🤷‍♂

Reply:
yup, better!